app/Activity.php

```diff
@@ -14,4 +14,35 @@ public function space() {
     public function user() {
         return $this->belongsTo(User::class);
     }
+
+    public function getFormattedCreatedAtAttribute() {
+        $date1 = strtotime($this->created_at);
+        $date2 = strtotime("now");
+
+        // Formulate the Difference between two dates
+        $diff = abs($date2 - $date1);
+
+        $years = floor($diff / (365*60*60*24));
```

Comment:
Can you add spaces between the multiplications, just like you did with the minuses? Cheers